contributing_to_docs/contributing.adoc

```diff
@@ -130,8 +130,7 @@ are released, the `master` branch is merged down to new or existing release
 branches. Here is the general naming scheme used in the branches:
 
 * `master` - This is our *working* branch.
-* `master-3` - OKD latest code; essentially, this is our *working*
-branch for the 3.x stream.
+* `master-3` - *working* branch for the 3.x stream.
```

Comment:
@vikram-redhat, can we drop `master-3` and just commit directly to `enterprise-3.11`?

Reply:
@kalexand-rh Not just yet. Two reasons: 

* I have a few 3.10/3.9 bugs that _may_ be getting exceptions. 
* OKD4 is not yet GA.

By themselves these might not be reasons enough to not switch, but taken together I think we should hold off.

Reply:
I think you're right. As long as you're ok with us CPing organic submissions back to `master-3`, it can wait.